determined-ai/determined
# build: set up shared-web submodule [DET-6961]

## Description

- move message component as an example




## Test Plan

- expect the build to fail with a bad submodule SHA
- check that we still
  - build the stories for shared modules
  - run and include the shared module tests





## Commentary (optional)

shared-web repo: https://github.com/determined-ai/shared-web




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
